### DISCUSSION TITLE
[Question] How to export the filtered version of the photo. onImageEditingComplete not triggered.

### DISCUSSION THREAD

**Discussion by umityilmazz** [hm21/pro_image_editor#296] [Q&A] [answered]
I'm trying to add a filter to the photo using FilterEditor.asset. How to export the filtered version of the photo. onImageEditingComplete not triggered.

**Comment by hm21** [comment, marked as answer]
You also need to set the `convertToUint8List` flag to `true`. Refer to this [example](https://github.com/hm21/pro_image_editor/blob/8a0dcd76ceaa11fc3e43cac8f065ebc0cdff5176/example/lib/pages/standalone_example.dart#L207-L221) for guidance on how to implement this.
